### ISSUE TITLE
Support reading in the IDE with screen readers

### ISSUE THREAD

**Issue by Qchristensen** [parallaxinc/PropellerIDE#56]
My organisation makes the free NVDA screen reader which is relied on by blind users around the world.  I have had a request from a user this morning who would like to use Propeller IDE but is having trouble as it is not accessible.  In trying it myself, I found that NVDA cannot read the text in the editor.

If the editor could be made accessible (often by using standard edit controls, or by implementing accessibility if using custom controls) that would make the program useable by blind users.
